node/npm/glide/tsconfig-base.json

```diff
@@ -0,0 +1,29 @@
+{
```

Comment:
can this extend `../tsconfig-base.json`?

Reply:
Will check, if so, actually i will delete this one completely and extend the commonjs and ecma tsconfig's directlly.
But - wont it be more right to use the tsconfig files of the general node folder in the build scripts of npm directly? If there's no known reasons to differentiate, we should use the same translators internally and externally, so we know every change in translation rules will happen everywhere and avoid the option of someone adding change in just one place mistakenly.